src/App.jsx

```diff
@@ -1,3 +1,16 @@
+import { useState } from "react";
+import { RecentThoughts } from "./components/RecentThoughts/RecentThoughts";
+import { CreateThought } from "./components/CreateThought/CreateThought";
+import { Header } from "./components/Header/Header";
+
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
+  const [items, setItems] = useState([]);
+
+  return (
+    <div className="content-wrapper">
+      <Header />
+      <CreateThought />
+      <RecentThoughts items={items} setItems={setItems} />
+    </div>
+  );
 };
```

Comment:
Great job making the app.jsx a "clean entrance". This is something I wanted to change with my project, thank you for the inspiration!